.travis.yml

```diff
@@ -1,35 +1,31 @@
 language: scala
```

Comment:
supernit: can you get rid of the fuzz here? looks like the only needed one is the last few lines.